__init__.py

```diff
@@ -109,6 +109,15 @@ def _find_entity(self, entity, domains):
                               "dev_name": entity})
         return ha_entity
 
+    # Routine for entiti availibility check
+    def _check_availability(self, ha_entity):
```

Comment:
in the future a doc string would be helpful for reviewers to understand the method at a glance

Reply:
Done

Reply:
Done